src/settings/SettingsSelectors.ts

```diff
@@ -10,9 +10,15 @@ The database related to a card is, at its start, the same as the one defined ins
 a user can modify the database that is used by a card with a new option inside the card itself.
  */
 export const getDatabase = (state: any, pageNumber:number, cardIndex:number) => {
-    let reportDatabase = state.dashboard.pages[pageNumber].reports[cardIndex].database
-    if (reportDatabase != undefined) {
+    if(state == undefined || pageNumber == undefined || cardIndex == undefined){
+        return "neo4j";
```

Comment:
Shouldn't be the default system? neo4j may not exist on EE deployments

Reply:
Good one. The problem is that we don't have that information about this point, at least, not persisted in the global state. That means that we'd have to fire off another query here to get the default system one I guess.

Do you think adding a TODO here would suffice for now? And we pick it up at the refactor where we introduce a global state database list?